question16/main.js

```diff
@@ -83,21 +82,57 @@ const createTitles = ({ articles }) => {
     const titleList = document.createElement('li');
     const articleLink = document.createElement('a');
     articleLink.href = "#";
-    articleLink.textContent = articleTitle; 
+    articleLink.textContent = articleTitle;
+
+    // Display new icon
+    const elapsedTime = getElapsedDays(articles[i]);
+    if (elapsedTime <= 3) {
```

Comment:
IMO
Since this would be a magic number, I felt that it should be explicitly stated as a constant if possible.

Reply:
Thank you for pointing it out! It is true! I have specified the magic number as a constant. Please confirm.

Latest commit : https://github.com/csakyo/homework/pull/30/commits/e507fac706dd73eee0409c7c71f3f465a3a636ce